eclair-core/src/test/scala/fr/acinq/eclair/blockchain/bitcoind/BitcoinCoreClientSpec.scala

```diff
@@ -961,4 +961,21 @@ class BitcoinCoreClientSpec extends TestKitBaseClass with BitcoindService with A
     assert(Btc(receivedAmount.values).toMilliBtc == amount)
   }
 
+  test("generate segwit change outputs") {
```

Comment:
Note that this test suite sets the `addresstype` and `changetype` to `bech32m` (see the fixture setup)